### PR TITLE
Add tests for coverage on main branch

### PR DESCRIPTION
**Pull request by sebbi08**
# Readiness checklist

-   [ ] I added/updated tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

**Comment by sebbi08** [comment]
We need to also publish the code coverage from the main branch to get a better comparison

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Coverage
+
+on:
+  push:
+    branches: [main]
```
Comment (jkoenig134) [inline .github/workflows/coverage.yml:5]: should also be published from release branches

**Comment by jkoenig134** [comment]
is this really necessary? We disabled tests on main a while ago to reduce overall runtime b/c everything is tested before on the branches itself..

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (jkoenig134) [reply .github/workflows/coverage.yml:5]: ah, right!

**Comment by sebbi08** [comment]
yes, Codecov needs a coverage report for the PR base commit

image

**Comment by jkoenig134** [comment]
> is this really necessary? We disabled tests on main a while ago to reduce overall runtime b/c everything is tested before on the branches itself..

Ok, now I see the Problem. We don't get information about lines changed. https://app.codecov.io/gh/nmshd/connector/pull/233

If this is really necessary I'd say enable the tests for main instead of creating a new workflow here.

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**